src/wolfio.c

```diff
@@ -1142,19 +1142,20 @@ int wolfIO_HttpProcessResponse(int sfd, const char** appStrList,
 
 int wolfIO_HttpBuildRequest(const char* reqType, const char* domainName,
     const char* path, int pathLen, int reqSz, const char* contentType,
-    byte* buf, int bufSize)
+    const char* exHdrs, byte* buf, int bufSize)
```

Comment:
This is a public function.
The prototype cannot be changed.
Add a wolfIO_HttpBuildRequest_ex API.

Reply:
Added wolfIO_HttpBuildRequest_ex